generators/tests/org/jetbrains/kotlin/generators/tests/GenerateTests.kt

```diff
@@ -1812,6 +1814,20 @@ fun main(args: Array<String>) {
             }
         }
 
+        testGroup("plugins/uast-kotlin-fir/tests", "plugins/uast-kotlin-fir/testData") {
+            testClass<AbstractFirUastDeclarationTest> {
+                model("declaration")
+                model("legacy")
```

Comment:
Looks like the `legacy` directory is copy-pasted from `uast-kotlin/testdata` directory. We can generate the fir tests over that directory instead of copy-pasting

Reply:
Yes, they were copy-n-pasted from `uast-kotlin/testData`. My intention was to not get overwhelmed by tests and by-products, and thus minimally(?) copy-n-pasted seemingly proper tests (e.g., no enum yet, no annotations yet, no facade yet, etc.).

But, maybe as long as things are isolated, it's good to add those early. I added `// IGNORE_FIR` directive to those test data (`.kt(s)`) in `uast-kotlin/testData` if not tested yet. The latest commit is mostly likely mechanical change/addition, and if added, FE 1.0 UAST plugin's render/log outputs.